index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header class="header">
+    <nav class="navbar">
+        <a href="#" class="nabar-brand"> My logo</a>
+        <i class="fa fa-bars" class="menu-icon"></i>
+        
+    </nav>
+    </header>
+
+    <section class="Mobile-view">
+        <div class="mobile-container">
+            <div class="info">
+                <p class="intro">Hey There. I'm alex.</p>
+                <p class="skill">I'm a Software Developer</p>
```

Comment:
- Hey, this is a subheading to the main heading of your page's content and as such should be from an h2-h6 tag. let's improve our semantic usage.